web_responsive/static/src/js/web_responsive.js

```diff
@@ -61,19 +62,43 @@ odoo.define('web_responsive', function(require) {
 
     var AppDrawer = Class.extend({
```

Comment:
Shouldn't this inherit from `Widget`?

Reply:
I'm almost positive I tried this as a widget originally, but I can't remember the problem. I don't think I want to expand this scope to that though TBH. 

Roadmap?

Reply:
Roadmap 👍 

If we can't do it is because we don't know or have time, but for sure it should be.